Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -99,6 +92,32 @@ theorem tensto_primeCounting : Tendsto π atTop atTop :=
 theorem prime_nth_prime (n : ℕ) : Prime (nth Prime n) :=
   nth_mem_of_infinite infinite_setOf_prime _
 
+@[simp]
+lemma primeCounting'_eq_zero_iff {n : ℕ} : n.primeCounting' = 0 ↔ n ≤ 2 := by
```

Comment:
It would be nice to have
```
lemma Nat.count_eq_zero {P : ℕ → Prop} [DecidablePred P] (h : ∃ n, P n)  {n : ℕ} :
    count P n = 0 ↔ n ≤ nth P 0 := by
```